executor/executor.go

```diff
@@ -371,6 +371,9 @@ func (e *CheckTableExec) Next(ctx context.Context, chk *chunk.Chunk) error {
 	defer func() { e.done = true }()
 	dbName := model.NewCIStr(e.ctx.GetSessionVars().CurrentDB)
 	for _, t := range e.tables {
+		if dbName.L == "" {
```

Comment:
Is this a bug-fix?

Reply:
Ye~
If use `e.ctx.(sqlexec.RestrictedSQLExecutor).ExecRestrictedSQL(e.ctx, "admin check table test.t1")` ,here `dbName.L` is "".

Reply:
Could we use `t.DBInfo.Name` directly?

Reply:
@crazycs520